src/posix/platform/netif.cpp

```diff
@@ -1251,6 +1251,15 @@ static void processNetifAddrEvent(otInstance *aInstance, struct rt_msghdr *rtm)
 
     addr6.sin6_family   = 0;
     netmask.sin6_family = 0;
+    // Skip processing RTM_NEWADDR or RTM_NEWMADDR after interface disabled
+    if (false == otIp6IsEnabled(aInstance))
```

Comment:
```suggestion
    if (!otIp6IsEnabled(aInstance))
```
Some code/style-checkers will flag it if `== false` or `== true` are used with boolean values.